File: searching for a life (algarithoms)/Linear_search.js

let data = [1,5,2,5,323,1,54,23,456]

function Linear_look(val,arr){
    for (let index = 0; index < arr.length; index++) {
        if (arr[index] == val) {
            return index
        }
    }
    return -1
}

console.log(Linear_look(2,data))
console.log(Linear_look(900,data))

File: searching for a life (algarithoms)/Linear_search.dart

int Linear_search(arr,key){
  for (var i = 0; i < arr.length; i++) {
    if (arr[i] == key) {
      return i;
    }
  }
  return -1;
}


void main() { 
  var data = [1,5,2,5,323,1,54,23,456]; 
  print(Linear_search(data,2));
  print(Linear_search(data,5));
  print(Linear_search(data,9999));
}